server/src/api/routing.ts

```diff
@@ -1,11 +1,11 @@
-const express = require('express');
-const { NotFoundError } = require('./errors');
-const logger = require('./logger');
-const playerRoutes = require('./modules/players/player.route');
-const deltaRoutes = require('./modules/deltas/delta.route');
-const recordRoutes = require('./modules/records/record.route');
-const competitionRoutes = require('./modules/competitions/competition.route');
-const groupRoutes = require('./modules/groups/group.route');
+import * as express from 'express';
+import * as logger from './logger';
+import { NotFoundError } from './errors';
+import { api as playerRoutes } from './modules/players/player.route';
```

Comment:
This is a bit strange, can't you export the correct name in the first place? `export { playerRoutes }`

Reply:
I was trying to keep naming the same as possible for the actual exports, but I can shift to the routes name anyways.